Email Report Extension task throwing error - File not found: /Index.js

Hi, 

As per our requirement we **need to send an email with Test Results** once the Automation test scripts are executed via Pipeline. So we have installed the "**Email Report Extension**" from marketplace and configured the SMPT service connection as well. After publishing the Automation test results, we are executing the Email Report Task in pipeline. But it is throwing the **File not found** Error.

Please help us to fix this issue.

Attaching the logs for your reference :

2025-02-28T05:32:32.0257798Z ##[section]Starting: EmailReportV2
2025-02-28T05:32:32.0263681Z ==============================================================================
2025-02-28T05:32:32.0263838Z Task         : Email Report
2025-02-28T05:32:32.0263952Z Description  : Send rich email report for test results and for tasks in this stage
2025-02-28T05:32:32.0264100Z Version      : 2.0.12
2025-02-28T05:32:32.0264209Z Author       : Microsoft Corporation
2025-02-28T05:32:32.0264324Z Help         : Sets 'EmailReportTask.EmailSent' variable to true if it sent email successfully. [Learn More](https://marketplace.visualstudio.com/items?itemName=epsteam.EmailReportExtension)
2025-02-28T05:32:32.0264557Z ==============================================================================
2025-02-28T05:32:32.0426487Z ##[error]File not found: '/home/vsts/work/_tasks/EmailReportV2_36fd41b1-8024-4ce9-a5a0-53c3e54ed105/2.0.12/Index.js'
2025-02-28T05:32:32.0437230Z ##[section]Finishing: EmailReportV2

--- a/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV1/task.dev.json
+++ b/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV1/task.dev.json
@@ -228,7 +228,7 @@
   "instanceNameFormat": "Send Email Report $(sendMailConditionConfig)",
   "execution": {
     "Node": {
-      "target": "Index.js"
+      "target": "index.js"
     }
   }
 }

--- a/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV1/task.json
+++ b/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV1/task.json
@@ -228,7 +228,7 @@
   "instanceNameFormat": "Send Email Report $(sendMailConditionConfig)",
   "execution": {
     "Node": {
-      "target": "Index.js"
+      "target": "index.js"
     }
   }
 }

--- a/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV2/task.dev.json
+++ b/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV2/task.dev.json
@@ -228,14 +228,14 @@
   "instanceNameFormat": "Send Email Report $(sendMailConditionConfig)",
   "execution": {
     "Node": {
-      "target": "Index.js"
+      "target": "index.js"
     },
     "Node16": {
-      "target": "Index.js",
+      "target": "index.js",
       "argumentFormat": ""
     },
     "Node20_1": {
-      "target": "Index.js",
+      "target": "index.js",
       "argumentFormat": ""
     }
   }

--- a/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV2/task.json
+++ b/Extensions/emailReportTask/Tasks/emailReportTask/emailReportTaskV2/task.json
@@ -228,14 +228,14 @@
   "instanceNameFormat": "Send Email Report $(sendMailConditionConfig)",
   "execution": {
     "Node": {
-      "target": "Index.js"
+      "target": "index.js"
     },
     "Node16": {
-      "target": "Index.js",
+      "target": "index.js",
       "argumentFormat": ""
     },
     "Node20_1": {
-      "target": "Index.js",
+      "target": "index.js",
       "argumentFormat": ""
     }
   }

--- a/Extensions/pullRequestInsightsTask/task.dev.json
+++ b/Extensions/pullRequestInsightsTask/task.dev.json
@@ -60,7 +60,7 @@
     ],
     "execution": {
         "Node": {
-            "target": "Index.js"
+            "target": "index.js"
         }
     }
 }

--- a/Extensions/pullRequestInsightsTask/task.json
+++ b/Extensions/pullRequestInsightsTask/task.json
@@ -60,7 +60,7 @@
     ],
     "execution": {
         "Node": {
-            "target": "Index.js"
+            "target": "index.js"
         }
     }
 }

--- a/Extensions/pullRequestInsightsTask/task.prod.json
+++ b/Extensions/pullRequestInsightsTask/task.prod.json
@@ -60,7 +60,7 @@
     ],
     "execution": {
         "Node": {
-            "target": "Index.js"
+            "target": "index.js"
         }
     }
 }

